LykkeApi2/Controllers/DepositsController.cs

```diff
@@ -118,27 +147,50 @@ public async Task<IActionResult> GetFxPaygateFee()
         [ProducesResponseType(typeof(FxPaygatePaymentUrlResponseModel), (int) HttpStatusCode.OK)]
         public async Task<IActionResult> PostFxPaygate([FromBody] FxPaygatePaymentUrlRequestModel input)
```

Comment:
It better to rename related methods and classes names to *link4pay* or semothing neutral like *payments*